### ISSUE TITLE
Search within a specific book

### ISSUE THREAD

**Issue by chaimkut** [Sefaria/Sefaria-Project#378]
I would like to be able to perform a query within a particular text. This could be accomplished with an advanced query ability (with a UX means of indicating the books to be searched); or at least as a first step, one could allow a "text:" operator which can be used to specify the intended corpus. The search API suggests using a regex search on the path, so that's certainly one way to implement.

Obviously providing an operator is more challenging because then the user has to correctly specify the text's name... though I see elsewhere on the site that you have some sort of canonicalization function (Bereshit ~ Bereshith ~ Genesis) which perhaps could be used here.

The same request for an "author:" operator would also be valid, though I think less universally useful.

**Comment by nahariyah** [comment]
YES!!! Please make this possible!!!! And if it already is please post instructions or a youtube video on how to do it.
I know so many people who want to do this and have had so many occasions myself!

**Comment by nsantacruz** [comment]
You can currently accomplish this by using the global search in the header and then using the filter menu to filter for the book you're interested in. I believe this use-case would be better solved by having some button in the sidebar to search within the current book. I'll add this to our feature request list.

**Comment by nahariyah** [comment]
Thank you for the instructions!!! and love that feature request!  It would
be nice if it could be further narrowed to parasha for example.  Often when
translating/studying chasidic text they'll cite a parsha for quote but not
cite the pasuk.  And sometimes they cite a book...eg psalms, or degel
M.Efraim or perhaps a section of Talmud.
Sefaria really shines when there is a quote with no citation!
Sefaria has made text study so much easier! love it!
Nahariyah
